pkg/user/tx_client.go

```diff
@@ -786,6 +801,11 @@ func (client *TxClient) DefaultAddress() sdktypes.AccAddress {
 
 func (client *TxClient) DefaultAccountName() string { return client.defaultAccount }
 
+// IsTrustedMode returns whether the client is operating in trusted mode.
+func (client *TxClient) IsTrustedMode() bool {
+	return client.trustedMode
+}
```

Comment:
I think it reads better if it's like this: 
```suggestion
func (client *TxClient) IsTrusting() bool {
	return client.trustedMode
}
```